packages/integrations/useDrauu/index.ts

```diff
@@ -0,0 +1,122 @@
+import { ref, watch } from 'vue-demi'
+import { createDrauu, Drauu, Options } from 'drauu'
+import { MaybeElementRef, unrefElement, createEventHook } from '@vueuse/core'
+import { tryOnScopeDispose, Fn } from '@vueuse/shared'
+
+export type UseDrauuOptions = Omit<Options, 'el'>
+
+/**
+ * Reactive drauu
+ *
+ * @see https://vueuse.org/useDrauu
+ * @param target The target svg element
+ * @param options Drauu Options
+ */
+export function useDrauu(target: MaybeElementRef, options?: UseDrauuOptions) {
+  const drauuInstance = ref<Drauu>()
+
+  let disposables: Fn[] = []
+
+  const onChangedHook = createEventHook<void>()
+  const onCanceledHook = createEventHook<void>()
+  const onComittedHook = createEventHook<void>()
+  const onStartHook = createEventHook<void>()
+  const onEndHook = createEventHook<void>()
+  const canUndo = ref(false)
+  const canRedo = ref(false)
+  const altPressed = ref(false)
+  const shiftPressed = ref(false)
+
+  const brush = {
```

Comment:
I feel like `brush `itself might be better to be a ref (so you can swap the entire brush)

Reply:
Hmm yeah I was having trouble figuring which way it should be. Because right now it is easy to hook up to a v-model for things like the brush size, but we aren't able to swap out brushes so I'm not sure what the best solution is.

Reply:
We have an extended `toRefs` utils that accept using passing an object ref. We could document that.

Reply:
Sounds good, will update

Reply:
Ok I updated the function to take a ref for the brush. I also created a new function that returns a reactive object with the brush properties so it is a bit easier to create brushes, let me know what you think of that. It worked out well in the demo and I was able to avoid using something like `toRefs`